src/shogun/lib/tag.h

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
BTW the filenames differ from what we usually do for shogun......
@lisitsyn can you comment on that?